modules/op-release-notes-1-6.adoc

```diff
@@ -0,0 +1,204 @@
+// (link:https://github.com/tektoncd/cli/pull/1443[#1443])
+
+* This update adds support for the triggers resources to work with `v1beta1` resources.
+
+// (link:https://github.com/tektoncd/cli/pull/1446[#1446], link:https://github.com/tektoncd/cli/pull/1449[#1449], link:https://github.com/tektoncd/cli/pull/1450[#1450], link:https://github.com/tektoncd/cli/pull/1454[#1454], link:https://github.com/tektoncd/cli/pull/1455[#1455])
+
+* This update adds an `ignore-running` option to the `tkn pipelinerun delete` and `tkn taskrun delete` commands.
+// (link:https://github.com/tektoncd/cli/pull/1445[#1445])
+
+* This update adds a `create` subcommand to the `tkn task` and `tkn clustertask` commands.
+// (link:https://github.com/tektoncd/cli/pull/1359[#1359])
+
+* With this update, if you use the `tkn pipelinerun delete --all` command, you can preserve runs that are younger than a specified number of minutes by using the new `--keep-since <minutes>` option.
+// (link:https://github.com/tektoncd/cli/pull/1392[#1392])
++
+.Example command to keep task runs that are less than five minutes old
+[source,terminal]
+----
+$ tkn pipelinerun delete -all --keep-since 5
+----
+
+* With this update, when you use the `tkn pipelinerun delete --all` command, you can use the new `--label <string>` option to filter the pipeline runs by label. Optionally, you can use the `--label` option with `=` and `==` as equality operators, or `!=` as an inequality operator. For example, the `tkn pipelinerun delete --all --label asdf` and  `tkn pipelinerun delete --all --label==asdf` commands both delete all the pipeline runs that have the `asdf` label.
+// (link:https://github.com/tektoncd/cli/pull/1402[#1402])
+
+* With this update, you can fetch the version of installed Tekton components from the config map or, if the config map is not present, from the deployment controller.
+//  (link:https://github.com/tektoncd/cli/pull/1393[#1393])
+
+[id="new-features-tekton-triggers-0-16-0-release-1-6_{context}"]
+=== Tekton Triggers
+
+* With this update, triggers support the `feature-flags` and `config-defaults` config map to configure feature flags and to set default values respectively.
+//  (link:https://github.com/tektoncd/triggers/pull/1182[#1182], link:https://github.com/tektoncd/triggers/pull/1110[#1110])
+
+* This update adds a new metric, `eventlistener_event_count`, that you can use to count events received by the `EventListener` resource.
+//  (link:https://github.com/tektoncd/triggers/pull/1160[#1160])
+
+* This update adds `v1beta1` Go API types. With this update, triggers now support the `v1beta1` API version.
++
+With the current release, the `v1alpha1` features are now deprecated and will be removed in a future release. Begin using `v1beta1` features instead.
+//  (link:https://github.com/tektoncd/triggers/pull/1103[#1103])
+
+[id="new-features-pipelines-operator-1-6_{context}"]
+=== {pipelines-title} Operator
```

Comment:
@VeereshAradhya Thanks for providing this information. To document release notes for these items, I need more information, such as a more complete description of the new feature and how the user can make use of it. This information must be similar in scope and detail to the other release notes visible here. Can you provide this information, link to a source for this information, such as a pull request or Jira item, or indicate who might be able to provide it?

Reply:
@rolfedh Fresh from today's stand up: PR for auto-prunning - https://github.com/tektoncd/operator/pull/410

Reply:
> Missing items
> 
> 1. Disabling `pipeline` service account
> 2. Updating feature flags using tektonconfig CR

@vdemeester @concaf @pradeepitm12 I cannot provide release notes for the items @VeereshAradhya mentions unless someone provides additional information.

Reply:
**DIsabling `pipeline` service account**

It is now possible to disable, cluster-wide, the creation of the `pipeline` service account. The rational for disabling `pipeline` builtin service account, is for security-concerned cluster-admin that do not want their user to have a slight privilege escalation as the `pipeline` comes with a SCC very similar to `anyuid`

**Updating feature-flags and configuration using the Operator's CRDs**

It is now possible to configure feature-flags and component configuration directly from the TektonConfig CR (and the per-component CR `TektonPipeline`, `TektonTriggers`, …). This allows user to customize their OpenShift Pipelines instances as well as enabling alpha features.

Reply:
** Optional `Timeouts` field on `PipelineRun` **

It is now possible to configure more timeouts on the PipelineRun. There is a timeout for the pipeline execution, per-task execution and for the finally tasks execution.

** `TaskRun` Pods are now in the "terminating" quota **

Pods generated by Tekton TaskRuns are now setting the `activeDeadlineSeconds` field on Pods, making them considered as Terminating for Kubernetes. This helps supporting ResourceQuota a bit better as now Tekton Pipeline's pod are considered terminating and thus can be using a specific scoped ResourceQuota for those.

** Add Configuration for Metrics Cardinality Simplification **

Metrics tags/labels type on taskrun/pipelinerun and task/pipeline can be eliminated metrics using configmap. Also, the type of metrics for duration - histogram or gauge/last value, can also be configured.

** Better LimitRange support **

Tekton now supports fully LimitRange by taking into account the `Min`, `Max`, `Default` and `DefaultRequest` fields to define a coherent Requests and Limits on the Pod.

** Alpha Features **

* Graceful PipelineRun Termination *

`PipelineRun` can now be stopped gracefully than the previous cancellation behavior that would stop the execution of all `TaskRun` directly and not running `finally` Task. This now adds the following `spec.status` values:

- `StoppedRunFinally` to stop the current tasks (let them complete) and then run the finally tasks
- `CancelledRunFinally` to cancel the current tasks (interupt any execution) and then run the finally tasks
- `Cancelled` to keep the same behavior as previous `PipelineRunCancelled` status.

The status "PipelineRunCancelled" is deprecated and replaced by "Cancelled" (it would be removed in v1). 

* Debug `TaskRun` with breakpoint onFailure *

It is possible to put a `TaskRun` into debug mode, that will pause the execution and allow the user to get into the specific step (inside the Pod) using, for example, `oc debug`.

* `onError` behaviour for Steps *

When `onError` is set to continue, the Step's exit code is recorded and made available to subsequent Steps via two new variables in Tasks. A Step with `onError` set to continue does not fail a `TaskRun` and continues executing the rest of the steps in a task. The default value to `onError` is `stopAndFail` to keep the current behavior.

* Implement implicit parameter resolution. *

Tasks can now accept more parameters than are actually used. Parameters can now be implicitly propagated to inlined specs (alpha feature flag must be enabled) - e.g. an inlined Task can access parameters of its parent PipelineRun without needing to explicitly define each param.

Reply:
To be added in ** Alpha Features **

* Skiping Strategies *

When Expressions are changing scope: in a future release (and today, if you enable the flag) a when stanza will guard only the Task it is attached to instead of the Task and all of its dependents. To guard both a Task and its dependents the when stanza will need to be attached to each dependent Task as well.

This should also be added to the Deprecation notice, as this change will roll out for all users of v1beta1 resources in 9 months, per our beta deprecation policy, and will be the default behaviour for When Expressions in any new apiVersions of Tekton's resources going forward. Thus it will most likely change in 1.8 or 1.9.